libs/payments/eligibility/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Lisa's [PR](https://github.com/mozilla/fxa/pull/15791) for the `CapabilityManager` library has an `eslintrc.json` file in her `capability` directory as does my [shared error library PR](https://github.com/mozilla/fxa/pull/15627).

My diff from running the [`nx g` command in the Ecosystem docs](https://mozilla.github.io/ecosystem-platform/reference/npm-scripts-and-nx#creating-a-library-in-nx) did not, and I manually added it.

@xlisachan or @julianpoy : Is there something missing from this command, or has something changed to the library generation config in the meantime?

Reply:
Ah nevermind; I see it was generated. I must have accidentally deleted it afterward. Thanks Lisa for the sanity check!

Reply:
Okay actually a related question here: while eslintrc.json was added, it was added originally with a `@nx/dependency-checks` rule below resulting in a Lint job error also below:

`eslintrc.json` addition I since removed after talking with @xlisachan who had previously chatted with Ben  -- @bbangert : What's this rule about, why is it getting automatically added, why don't we want it to be added, and can we adjust the library gen config to avoid adding it if we don't need it?:
```
{
"files": ["*.json"],
  "parser": "jsonc-eslint-parser",
  "rules": {
    "@nx/dependency-checks": "error"
  }
}
```

Linting error with above eslint rule applied:
```
bdanforth ~/Projects/fxa (FXA-7582) $ npx nx run payments-eligibility:lint

> nx run payments-eligibility:lint


Linting "payments-eligibility"...
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <5.1.0

YOUR TYPESCRIPT VERSION: 5.2.2

Please only submit bug reports when using the officially supported version.

=============

/Users/bdanforth/Projects/fxa/libs/payments/eligibility/package.json
  1:1  error  Definition for rule '@nx/dependency-checks' was not found  @nx/dependency-checks

✖ 1 problem (1 error, 0 warnings)

Lint errors found in the listed files.


 ———————————————————————————————————————————————————————————————————————————

 >  NX   Ran target lint for project payments-eligibility (1s)
 
    ✖    1/1 failed
    ✔    0/1 succeeded [0 read from cache]
 
   View structured, searchable error logs at https://nx.app/runs/brWydsctWx
```

Reply:
FWIW answering this question shouldn't block the PR IMO as Lisa previously discussed this with Ben for her PR (that was already merged) and it's a matter of understanding why. I plan to ask Ben when he returns tomorrow (and read a bit more about it beforehand).